[ADD] helpdesk: doc about channels options (#646)
odoo/documentation-user: helpdesk/overview/receiving_tickets.rst

=======================
Start Receiving Tickets
=======================

Offering a variety of channels from where your customers can contact you grants them flexibility,
and the right to choose the best one for themselves. And in order to make sure inquiries across

> all channels get addressed, it is essencial to have a solution where all interactions come together

Reviewer: ```suggestion
all channels get addressed, it is essential to have a solution where all interactions come together
```

I still think the other suggestions I made previously are grammatically better.

> in one place.

Author: changed

Channels options to submit tickets
==================================

Go to :menuselection:`Helpdesk --> Configuration --> Helpdesk Teams`, and enable the following

> features as you want them to be available to your users.

Author: I could, but I do not see the need. It can be straightforward, as it also flows well with what comes next (:

> features as you want them to be available to your users.

Reviewer: The use would be to explain people why they do that ^^

> features as you want them to be available to your users.

Author: oh I missed your last answer, je suis désolé (🥇 for my french improvement haha)
Yeah, but I explain that following the menuselection: enable them as you want them to be available to your users o/o/

.. image:: media/channels_options.png
   :align: center
   :alt: View of a helpdesk teams setting page emphasizing the channels options in Odoo Helpdesk

Email Alias
-----------

| Let your customers submit tickets by sending an email to your support email address. The subject
  line of the email becomes the title of the ticket, and the email content is shown on the Chatter.

> | Select *Configure domain name* to be redirected to *Settings*, and from there, enable *External
>   Email Servers* to determine or change your *Alias Domain*.

Reviewer: ```suggestion
| Select *Configure domain name* to be redirected to *Settings* and, from there, enable *External
  Email Servers* to determine or change your *Alias Domain*.
```

.. image:: media/channels_emailalias.png
   :align: center
   :height: 250
   :alt: View of the settings page of a helpdesk team emphasizing the email alias feature
         in Odoo Helpdesk

.. note::
   Using your own email server is required to send and receive emails in Odoo Community and
   Enterprise. Online users benefit from a ready-to-use email server.

Website Form
------------

| Allow your customers to submit a ticket by filling in a form through your website.
| Once the feature is activated, get redirected to your website by clicking on *Go to Website*.

.. image:: media/go_to_website.png
   :align: center
   :alt: View of the settings page of a helpdesk team emphasizing the Go to Website button in
         Odoo Helpdesk

> From the website page, customize the form as you like, then choose to publish it by clicking
> on *Unpublished*.

Reviewer: ```suggestion
From the website page, customize the form as you like. Then, publish it by clicking
on *Unpublished*.
```

Author: I will keep it as one line (:

Reviewer 2: I prefer his way. You can keep it as one line but choose to needs to be removed.

Author: ok

.. image:: media/submit_a_ticket_form.png
   :align: center
   :alt: View of the website form to submit a ticket for Odoo Helpdesk

Live Chat
---------

| Through live interactions with your website visitors, helpdesk tickets can be instantly created
  and redirected to the right person.

> | Click on the name of your helpdesk team - for the example below: *Customer Care* -
>   and :doc:`set up your channel <../../livechat/overview/get_started>`.

Reviewer: ```suggestion
| Click on your helpdesk team's name - for the example below: *Customer Care* -
  and :doc:`set up your channel <../../livechat/overview/get_started>`.
```

.. image:: media/live_chat.png
   :align: center
   :alt: View of the settings page of a helpdesk team emphasizing the live chat features and links
         in Odoo Helpdesk

> Now, your operators can create tickets  by using the
> :doc:`command <../../livechat/overview/responses>` */helpdesk (subject_of_ticket)*.

Reviewer: ```suggestion
Now, your operators can create tickets by using the
:doc:`command <../../livechat/overview/responses>` */helpdesk (subject_of_ticket)*.
```
